src/main/scala/cromwell/engine/db/LiquibaseUtils.scala

```diff
@@ -0,0 +1,67 @@
+package cromwell.engine.db
+
+import java.sql.Connection
+
+import liquibase.database.DatabaseConnection
+import liquibase.database.jvm.{HsqlConnection, JdbcConnection}
+import liquibase.resource.{ClassLoaderResourceAccessor, ResourceAccessor}
+import liquibase.{Contexts, LabelExpression, Liquibase}
+import org.hsqldb.persist.HsqlDatabaseProperties
+
+import scala.collection.JavaConverters._
+
+object LiquibaseUtils {
+  val ChangeLogDir = "src/main/migrations/"
+  val DefaultChangeLog = "changelog.xml"
+  val DefaultResourceAccessor: ResourceAccessor = new ClassLoaderResourceAccessor()
```

Comment:
This is only used in one spot in the code, which is only executed once at startup.  Defining it here means it can never be GC'd so maybe an inlining would be preferable.

The two constants below are used in two spots, not sure if it's worth inlining those.